### DISCUSSION TITLE
Sensor specific zeek/suricata roles

### DISCUSSION THREAD

**Discussion by SliuzasLukas** [Security-Onion-Solutions/securityonion#11888] [Unsupported Versions] [answered]
How can a centralized manager efficiently distribute a combination of rules to various sensors within an IDS, where certain sensors may require a mix of Zeek and Suricata rules, while others may only need Zeek rules or a different combination?

**Comment by dougburks** [comment, marked as answer]
> How can a centralized manager efficiently distribute a combination of rules to various sensors within an IDS

The manager distributes the same Suricata ruleset to all sensors. If you want a particular sensor to not alert on a particular rule, then you can create a suppression for the subnet that the sensor is monitoring:
https://docs.securityonion.net/en/2.4/managing-alerts.html#suppress

>  where certain sensors may require a mix of Zeek and Suricata rules, while others may only need Zeek rules or a different combination?

Zeek doesn't really run rules in the same way that Suricata does. Are you saying that you want some sensors to not run Suricata at all? If so, you can disable Suricata on a particular sensor:
https://docs.securityonion.net/en/2.4/suricata.html#disabling
